### ISSUE TITLE
Brick, need help!

### ISSUE THREAD

**Issue by DeniZDobynda** [pwn20wndstuff/Undecimus#593]
Guys, I need help.

Before jailbreak, I did backup in iTunes.
Jailbreak was a success, though needed 3 reboots. It worked great, pretty stable.
Than I added another repo to Cydia, this one: cydia.angelxwind.net/
Springboard reloaded, but it took 3 times longer as usual, the phone didn't answered, so I did hard reload.
As soon as it turned on, jailbreak was gone. Nothing from Cydia would work, as well as Cydia itself. The apps wouldn't open.
Then I've decided to do Jailbreak a couple more times, but it never got to a working point.
So I've restored the phone with a backup. All went well, the backup was at place, but then the phone never loaded.
It passed the first Apple logo, went to darker mode and is there for more than an hour.
It doesn't help, to press and hold volume up and/or down with power button. The phone isn't responding.
The battery state was at a level 75 during the restore process. I hope in a few days phone will eventually turn off.

Can you, guys, help me? How can I get my phone to work again?

p.s.: I know the procedure to load to DFU and next steps, but it isn't an option for now. Phone is stuck and not responding to any holding combinations

EDIT: I have iPhone X, iOS 12.1.2
          I used the app from Ignition site

**Comment by zyan910** [comment]
Restoring the device which is jailbroken will end up in a bootloop, afaik. If the 12.1.1 b3 is signed, you can restore to that

**Comment by Marinamoina** [comment]
Press: volume up, vol down an than immediately hold power. This is a hard reset. All must be fast.dont hold buttons same time, all after one other. The repo is not the reason, I have it too.
If rebooted, delete uncover.
Go to home screen, hold power till slider appears. Hold home button until slider disappears, RAM is clear than.
Restart iPhone.
Side load uncover (latest version) again. Accept the trust of the app in settings.
Activate air plane mode, deactivate lock code and deactivate all Siri settings.
Close all app in app switcher.
Open unc0ver BUT WAIT FOR A MINUTE BEFORE PRESSING JAILBREAK. The app needs to load a little bit.
After one or two minutes, press jailbreak.
GOOD LUCK!! Tell us, if it worked.

**Comment by DeniZDobynda** [comment]
Guys, 
Thanks for help, I successfully recovered my iPhone.
That may be, that the whole problem was caused by taking phone of the cable during backup recovery.

Anyway, here's my steps:

1. Download iOS 12.1.1 beta 3
2. Put iPhone in recovery mode
3. Restore it with iTunes
3.1. Discover a few times Error 9. Flash it in DFU, also Error 9.
3.2. Try to restore 12.1.2 beta, same Error.
4. Reboot Mac
5. Flash 12.1.1 beta 3 in DFU, hope  still signs it
Done.
The phone is now working, next step: Jailbreak. (Cydia Impactor)

Thanks for help
